.github/workflows/ci.yml

```diff
@@ -9,50 +9,68 @@ on:
 jobs:
   xcode-build:
     name: Xcode Build
-    runs-on: macOS-latest
+    runs-on: macOS-10.15
     strategy:
       matrix:
-        platform: ['iOS_14', 'iOS_13', 'iOS_12']
+        platform: ['iOS_14', 'iOS_13']
```

Comment:
We now test the sample app in iOS 13 and 14 (smaller test matrix with the snapshot tests) and iOS 12 is tested with SPM.